crc32_benchmark.php

```diff
@@ -97,10 +101,10 @@ function test($crc, $chunk_size)
 }
 
 foreach (array(256, 4096, 1048576, 16777216) as $chunk_size) {
-    test(new CRC32_PHP(CRC32::CASTAGNOLI), $chunk_size);
-    test(new CRC32_PHP4(CRC32::CASTAGNOLI), $chunk_size);
+    test(new PHP(CRC32::CASTAGNOLI), $chunk_size);
```

Comment:
To be more explicit, would it be better to say

```
new Google\CRC32\PHP(...)
```

Reply:
Generally we import classes at the top of the file ([`use`](https://github.com/google/php-crc32/pull/3/files/ac50c442901051cab7c0efe93e90c272e32859f8#diff-af3a95411dd2d52f558602a2ee0591b8R23)) statement. Would you prefer using the fully-qualified name in this file?

Reply:
Happy to defer to you on PHP best practices.